### PR TITLE
User/yreda/reorder compilation and python script fixes

### PR DESCRIPTION
**Pull request by younesr1**
the reordering fixes a compilation error i get when building on the movo itself
the python changes fix import errors seen. likely needed bc python 3 imports are different from python 2

### CODE REVIEW

File: movo_robot/movo_bringup/launch/movo1.launch [hunk 1]
```diff
@@ -11,12 +11,9 @@
     <node pkg="si_utils" type="timed_roslaunch"
         args="8 movo_bringup movo_sensors.launch local:=false"
         name="sensors_bringup" output="screen"/>
-        
-     <!-- Platform Manipulation -->
-    <node pkg="si_utils" type="timed_roslaunch"
-        args="12 movo_bringup movo_manipulation.launch local:=false"
-        name="manipulation_bringup" output="screen"/> 
 
+    <!-- YOUNES TODO DO NOT MERGE PLACE ARM BRINGUP BACK -->
```
Comment (younesr1) [inline movo_robot/movo_bringup/launch/movo1.launch]: ack of this todo. will not merge until i undo

Comment (younesr1) [reply movo_robot/movo_bringup/launch/movo1.launch]: moved this to a seperate pr

**Review by FelixKHWong — APPROVED** [review]
I remember these changes. They are required to build correct? Also what was the justification against changing the python path to include "movo"? I just want the reason here in case so we can reference it later if it gets annoying to keep doing adding the folder names.

File: movo_common/movo_ros/src/movo/movo_comm.py [hunk 2]
```diff
@@ -33,17 +33,17 @@
 
  \Platform: Linux/ROS Indigo
```
Comment (FelixKHWong) [inline movo_common/movo_ros/src/movo/movo_comm.py:34]: Should this be changed? Applies to all files here

Comment (younesr1) [reply movo_common/movo_ros/src/movo/movo_comm.py:34]: this pr only deals with fixing python import errors and reorder a the cmake line. feel free to make a ticket for updating all comments from indigo to noetic

**Comment by younesr1** [comment]
> I remember these changes. They are required to build correct? Also what was the justification against changing the python path to include "movo"? I just want the reason here in case so we can reference it later if it gets annoying to keep doing adding the folder names.

i think its bc ROS doesnt retain your original folder structure when u compile. the scripts are sym linked in the devel folder in a different structure i believe. anyway heres an article that does the same import work around https://roboticsbackend.com/ros-import-python-module-from-another-package/

**Pull request merged**